embedded-workshop-book/src/SUMMARY.md

```diff
@@ -0,0 +1,49 @@
+# Summary
+
+- [Intro](./intro.md)
```

Comment:
nitpick: the almost empty page looks very Work In Progress-y, I'd rather comment it out until we have content for it